captum/optim/_utils/datasets.py

```diff
@@ -0,0 +1,53 @@
+import torch
+

```

Comment:
This looks image specific. Perhaps, it would be good to call it image_dataset.py ?

Reply:
Yeah, it's image specific so I've renamed it to image_dataset.py !